cmd/demo/README.md

```diff
@@ -0,0 +1,62 @@
+## About
```

Comment:
The `demo` folder contains some scripts, this should be moved to the root of the project OR in the root folder, you can create a folder named `scripts` where you will move the `demo` folder.

Reply:
The demo file itself is a `scripts` folder, it only consists of scripts.
I think its location is suitable to be under `cmd` along with the `connector` directory for the binary itself